### DISCUSSION TITLE
A QUESTION ABOUT CHUNKS BEING DELETED WITHOUT MY CONSENT

### DISCUSSION THREAD

**Discussion by MateusRestier** [Jikoo/Regionerator#252] [Q&A]
I'm making a server to play with my friends, I would like to know if this plugin could delete some chunks that we will miss, I don't really understand how flags work, if I build something far away, and don't visit it for a long time, Is there a chance that this chunk will be deleted?

**Comment by MateusRestier** [comment]
@Jikoo

**Comment by Jikoo** [comment]
Chunks are automatically protected for a time when players are around, but you can [manually create permanent flags](https://github.com/Jikoo/Regionerator/wiki/Commands#flagging-parameters) or protect them with a [supported protection plugin](https://github.com/Jikoo/Regionerator/wiki/Hooks#supported-plugins). If your area is not protected and not visited, yes, Regionerator would attempt to delete it.
